src/components/organisms/Accordion/Accordion/Accordion.test.tsx

```diff
@@ -3,7 +3,13 @@ import { render } from '@testing-library/react';
 import { Accordion } from '..';
 
 describe('<Accordion />', () => {
-  it('should render aria-label', () => {
+  it('should render standard attributes but not require aria-label', () => {
```

Comment:
Could we update this to just:
```js
it('should render supplied HTML attributes')
```
as the test is not testing `aria-label` is not required but just that `title` is propagated to the render ouput 💅🏻